gems/quilt_rails/lib/quilt_rails/configuration.rb

```diff
@@ -13,6 +13,7 @@ def initialize
       self.react_server_protocol = ENV['REACT_SERVER_PROTOCOL'] || "http"
       self.logger                = ::Logger.new($stdout)
       self.mount                 = true
+      self.integration_test_error = true
```

Comment:
I think I'd rather see this called like `allow_integration_test` and default to false, since I think that is a bit clearer than `integration_test_error`